### ISSUE TITLE
v2 Layer - restrictScroll prop

### ISSUE THREAD

**Issue by atanasster** [grommet/grommet#1865]
`Warning: React does not recognize the `restrictScroll` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `restrictscroll` instead. If you accidentally passed it from a parent component, remove it from the DOM element.`
```
in div (created by FocusedContainer)
    in FocusedContainer (created by LayerContainer)
    in LayerContainer (created by LayerContainer)
    in LayerContainer (created by Layer)
    in Layer (created by Layer)
    in Layer
```

**Comment by ericsoderberghp** [comment]
Can you provide some guidance on how to reproduce this? I haven't been able to and I can't see how the code would be trying to put `restrictScroll` into the DOM.

**Comment by atanasster** [comment]
This was fixed in #1901 
`delete rest.restrictScroll;`
unfortunately the issue was not mentioned and I didnt catch the fix on my end
